docker/tp2-base/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker image rm tp2-base
+docker build -t tp2-base .
```

Comment:
Formatting, new line at end of file
```suggestion
docker build -t tp2-base .

```